scripts/descriptions.json

```diff
@@ -42,94 +42,91 @@
   },
   "codecov.notify.countdown": {
-      "desc": "description: "
+      "desc": "No longer used"
   },
   "codecov.notify.delay": {
-      "desc": "description: "
+      "desc": "No longer used"
   },
   "codecov.notify.wait_for_ci": {
-      "desc": " True,  False,  yes,  no,  on,  off"
+      "desc": "Wait for CI to finish to send notifictionas. More info: https://docs.codecov.com/docs/codecovyml-reference#codecovnotifywait_for_ci. Enum: True,  False,  yes,  no,  on,  off"
   },
   "codecov.notify.require_ci_to_pass": {
-      "desc": " True,  False,  yes,  no,  on,  off"
+      "desc": "Wait for all CI statuses to pass. More info: https://docs.codecov.com/docs/codecovyml-reference#codecovrequire_ci_to_pass. Enum: True,  False,  yes,  no,  on,  off"
   },
   "codecov.notify": {
       "desc": "Configure how Codecov sends a PR comment"
   },
   "codecov.ui.hide_density": {
-      "desc": "description: "
+      "desc": "No longer used"
   },
   "codecov.ui.hide_complexity": {
-      "desc": "description: "
+      "desc": "No longer used"
   },
   "codecov.ui.hide_contexual": {
-      "desc": " True,  False,  yes,  no,  on,  off"
+      "desc": "No longer used"
   },
   "codecov.ui.hide_sunburst": {
-      "desc": " True,  False,  yes,  no,  on,  off"
+      "desc": "No longer used"
   },
   "codecov.ui.hide_search": {
-      "desc": " True,  False,  yes,  no,  on,  off"
+      "desc": "No longer used"
   },
   "codecov.ui": {
-      "desc": "description: "
+      "desc": "No longer used"
   },
   "codecov": {
-      "desc": "description: Configure general codecov settings. More info: https://docs.codecov.com/docs/codecov-yaml"
+      "desc": "Configure general codecov settings. More info: https://docs.codecov.com/docs/codecov-yaml"
   },
   "coverage.precision": {
-      "desc": "min: 0, description: "
-  },
-  "coverage.precision.max": {
-      "desc": 99
+      "desc": "What precision do you want the coverage value to be. More info: https://docs.codecov.com/docs/codecovyml-reference#coverageprecision"
   },
   "coverage.round": {
-      "desc": " down,  up,  nearest, description: "
+      "desc": "Configure how you want Codecov to round your coverage numbers. Enum: down,  up,  nearest"
   },
   "coverage.range": {
-      "desc": "maxlength: 2, description: "
+      "desc": "Configure how you want Codecov to color your coverage. Coverage below the first number will be red, coverage between each number yellow, and coverage above the second number will be green. Format: xx...yy "
   },
   "coverage.notify.irc.keysrules": {
-      "desc": "pattern: ^[\\w\\-\\.]+$"
+      "desc": "This is a regular expression for setting Key Rules as a string validated with the pattern: ^[\\w\\-\\.]+$"
   },
   "coverage.notify.irc.valuesrules.password": {
-      "desc": "nullable: True"
+      "desc": "Authenticate to an IRC channel. nullable: True"
   },
   "coverage.notify.irc.valuesrules.url": {
-      "desc": "nullable: True"
+      "desc": "Unneeded."
```

Comment:
If this is "unneeded", where do I tell Codecov how to connect to my IRC server?

Reply:
Joe and I spoke about this this morning and confirmed URL is not needed